src/utils.ts

```diff
@@ -4,14 +4,14 @@ import { QueryObject, parseQuery, stringifyQuery } from './query'
 import { decode } from './encoding'
 
 export function hasProtocol (inputStr: string): boolean {
-  return /^\w+:\/\//.test(inputStr)
+  return /^\w+:\/\/.+/.test(inputStr)
 }
 
 export function isURL (inputStr: string, acceptProtocolRelative = false): boolean {
   if (!acceptProtocolRelative) {
     return hasProtocol(inputStr)
   }
-  return /^(\w+:)?\/\//.test(inputStr)
+  return /^(\w+:)?\/\/.+/.test(inputStr)
```

Comment:
`///` is still wrong...

Reply:
Thoughts welcome. `///` isn't valid but `file:///` is. We could leave it inaccurate (as we are not doing validation of rest of string in any case). 🤷‍♂️

Reply:
We may change logic to `hasProtocol() || (acceptProtocolRelative && /\/\/[^/]+`/)` 

Also would be nice finding some reference on chrome why `//` is blocked

Reply:
Would work:
```suggestion
  return /^(\w+:)\/\/.+/.test(inputStr) || /^\/\/[^/]+/.test(inputStr)
```

Reply:
Isn't same part same as hasProtocol? :D